zero/gk_geometry_tok.c

```diff
@@ -129,6 +130,7 @@ gkyl_gk_geometry_tok_new(struct gkyl_gk_geometry_inp *geometry_inp)
   up->ref_count = gkyl_ref_count_init(gkyl_gk_geometry_free);
   up->on_dev = up; // CPU eqn obj points to itself
 
+  gkyl_tok_geo_release(geo);
```

Comment:
@akashukla is this the bug that @Maxwell-Rosen found in the mirror branch? is there a similar fix for mirror_geo?